tests/api/test_account.py

```diff
@@ -924,17 +928,54 @@ def test_address_update_mutation(
     assert address_obj.city == graphql_address_data['city']
 
 
+def test_customer_update_own_address(
+        user_api_client, customer_user, graphql_address_data):
+    query = ADDRESS_UPDATE_MUTATION
+    address_obj = customer_user.addresses.first()
+    address_data = graphql_address_data
+    address_data['city'] = 'Poznań'
+    assert address_data['city'] != address_obj.city
+
+    variables = {
+        'addressId': graphene.Node.to_global_id('Address', address_obj.id),
+        'address': address_data}
+    response = user_api_client.post_graphql(query, variables)
+    content = get_graphql_content(response)
+    data = content['data']['addressUpdate']
+    assert data['address']['city'] == address_data['city']
+    address_obj.refresh_from_db()
```

Comment:
Refreshing the `city` field should be enough?

Reply:
Yeah, but again, this solution is acceptable.